src/v1.rs

```diff
@@ -161,7 +161,8 @@ mod tests {
         let context = UuidContext::new(0);
 
         {
-            let uuid = Uuid::new_v1(&context, time, time_fraction, &node).unwrap();
+            let uuid =
```

Comment:
is this from latest fmt? looks messed up

Reply:
yes its the latest nightly fmt...